packages/apps/composer-app/src/playwright/app-manager.ts

```diff
@@ -33,7 +33,7 @@ export class AppManager {
     this._inIframe = inIframe;
   }
 
-  async init() {
+  async init(dontDismissFirstRun?: boolean) {
```

Comment:
nit: make options an object so it's easier to add more options without changing existing code